app/src/main/AndroidManifest.xml

```diff
@@ -9,16 +9,19 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
+        <activity android:name=".Devices"></activity>
         <activity
-            android:name="com.CSCI4320.journal_app.ui.login.LoginActivity"
+            android:name=".ui.login.LoginActivity"
```

Comment:
we should definitely maintain our package name as this is how the APK actually finds our app.